### PR TITLE
Increase process timeout period for sandboxed test cases

### PR DESCRIPTION
**Pull request by Kr1sso**
Some of our tests are timing out and failing our CI pipelines with runtimes slightly above 10 minutes. This PR pumps the timeout as a measure to not fail tests on slight deviations.

### CODE REVIEW

File: Public/Src/Engine/Processes/SandboxedProcess.cs [hunk 1]
```diff
@@ -81,11 +81,11 @@ internal SandboxedProcess(SandboxedProcessInfo info)
                 s_binaryPaths = new BinaryPaths(); // this can take a while; performs I/O
             }
 
-            // If unspecified make the injection timeout 10 mins. Also, make it no less than 10 mins.
-            m_timeoutMins = info.Timeout.HasValue ? ((uint)info.Timeout.Value.TotalMinutes) : 10;
-            if (m_timeoutMins < 10)
+            // If unspecified make the injection timeout 15 mins. Also, make it no less than 15 mins.
+            m_timeoutMins = info.Timeout.HasValue ? ((uint)info.Timeout.Value.TotalMinutes) : 15;
+            if (m_timeoutMins < 15)
```
Comment (almiliMSFT) [inline Public/Src/Engine/Processes/SandboxedProcess.cs]: can you use `DefaultProcessTimeout` from `UnSandboxedProcess.cs`? (or the other way around)

File: Public/Src/Utilities/Configuration/Mutable/SandboxConfiguration.cs [hunk 2]
```diff
@@ -16,7 +16,7 @@ public SandboxConfiguration()
             m_unsafeSandboxConfig = new UnsafeSandboxConfiguration();
 
             FailUnexpectedFileAccesses = true;
-            DefaultTimeout = 10 * 60 * 1000;
+            DefaultTimeout = 15 * 60 * 1000;
```
Comment (almiliMSFT) [inline Public/Src/Utilities/Configuration/Mutable/SandboxConfiguration.cs]: This is going to change the default timeout for all pips in all builds, right?  If so, I think this default value is also mentioned in the help text, so it should be updated there as well

File: Public/Src/Utilities/Utilities/VmCommandProxy/VmInitializer.cs [hunk 3]
```diff
@@ -25,7 +25,7 @@ public class VmInitializer
         /// <remarks>
         /// According to CB team, VM initialization roughly takes 2-3 minutes.
         /// </remarks>
-        private const int InitVmTimeoutInMinute = 10;
+        private const int InitVmTimeoutInMinute = 15;
```
Comment (almiliMSFT) [inline Public/Src/Utilities/Utilities/VmCommandProxy/VmInitializer.cs]: Is this related?  It looks to me that it is not

Comment (Kr1sso) [reply Public/Src/Utilities/Utilities/VmCommandProxy/VmInitializer.cs]: Good catch, search replace issue.

File: Public/Src/Utilities/Configuration/Mutable/SandboxConfiguration.cs (hunk 2 above)
Comment (Kr1sso) [reply Public/Src/Utilities/Configuration/Mutable/SandboxConfiguration.cs]: Help text is just a generic message explaining the value semantics.
